Source/Core/DolphinQt/Config/ControllersWindow.cpp

```diff
@@ -373,7 +373,7 @@ void ControllersWindow::OnWiimoteRefreshPressed()
 
 void ControllersWindow::OnEmulationStateChanged(bool running)
 {
-  if (!SConfig::GetInstance().bWii || NetPlay::IsNetPlayRunning())
```

Comment:
Not sure I understand what this does... So if the user starts a GameCube game then starts a Wii game, the options won't be enabled?

Reply:
I'm actually going to refactor this PR, as this code doesn't need to exist anymore since the controllers dialog is disabled while NetPlay is running.